doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -425,11 +422,34 @@ Response code:
 * Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
 
 + Parameters
-    + limit: 20 (optional, number)
-    + offset: 20 (optional, number)
-    + type: Room (optional, string)
-    + id: Bcn_Welt (optional, string)
-    + idPattern: (optional, string)
+    + id: Boe_Idearium (optional, string) - A comma separated list of elements.
+    Retrieve entities which ID match one of the elements in the list.
+    
+    + type: Room (optional, string) -  comma separated list of elements.
+    Retrieve entities which type match one of the elements in the list.
+
+    + idPattern (optional, string) - A correctly formated regular expression.
+    Retrieve entities which ID matches the regular expression.
+    
+    + q (optional, string) - A query expression, composed of a list of statements separated by <code>;</code>,
+    i.e. q=statement;statements;statement. See [Simple Query Language specification](#simple_query_language). 
+    
+    + geometry (optional, string) - Defines a geografical area so only the entities located in
+    that area matches the query. See Simple Query Language specification. 
```

Comment:
You should use `[Simple Query Language specification](#simple_query_language)` also here.